### PR TITLE
PEP 587: Clarify relationship with PEP 432

### PR DESCRIPTION
**Pull request by vstinner**

### CODE REVIEW

**Comment by vstinner** [comment]
@ncoghlan: Would you mind to review this PR? It should be the minimum change for what we discussed (in private).

File: pep-0587.rst [hunk 1]
```diff
@@ -29,6 +29,12 @@ The C API of Python 3.7 Initialization takes ``wchar_t*`` strings as
 input whereas the Python filesystem encoding is set during the
 initialization.
 
+This PEP is a partial implementation of PEP 432 which is the overall
+design.  New fields can be added later to ``PyConfig`` structure to
+finish the implementation of the PEP 432 (add a new partial
+initialization which allows to configure Python using Python objects to
+finish the full initializatio).
```
Comment (encukou) [inline pep-0587.rst]:
```suggestion
finish the full initialization).
```

**Comment by ncoghlan** [comment]
Thanks @vstinner. I'll put together a corresponding update to PEP 432 in the next day or two.

**Comment by ncoghlan** [comment]
Larger update to PEP 432 to account for the extraction of PEP 587: https://github.com/python/peps/pull/965

**Pull request merged**